doc/examples/transform/plot_masked_register_translation.py

```diff
@@ -0,0 +1,68 @@
+"""
+===================================
+Masked Normalized Cross-Correlation
+===================================
+
+In this example, we use the masked normalized cross-correlation to identify the relative shift
+between two similar images containing invalid data.
+
+In this case, the images cannot simply be masked before computing the cross-correlation, 
+as the masks will influence the computation. The influence of the masks must be removed from
+the cross-correlation, as is described in [1]_.
+
+In this example, we register the translation between two images. However, one of the 
+images has about 25% of the pixels which are corrupted.
+
+.. [1] D. Padfield, "Masked object registration in the Fourier domain" 
+       IEEE Transactions on Image Processing (2012). :DOI:`10.1109/TIP.2011.2181402`
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage import data
+from skimage.feature import masked_register_translation
+from scipy.ndimage import fourier_shift
+
+image = data.camera()
+shift = (-22, 13)
+
+# Define areas of the image which are invalid
+# Probability of an invalid pixel is 25%
+# This could be due to a faulty detector, or edges that
+# are not affected by translation (e.g. moving object in a window)
+# See reference paper for more examples
+corruped_pixels = np.random.choice([False, True], size = image.shape, p = [0.25, 0.75])
+
+# The shift corresponds to the pixel offset relative to the reference image
+offset_image = fourier_shift(np.fft.fftn(image), shift)
+offset_image = np.fft.ifftn(offset_image)
+offset_image *= corruped_pixels
+print("Known offset (y, x): {}".format(shift))
+
+# Determine what the mask is based on which pixels are invalid
+mask = np.logical_not(np.isclose(offset_image, 0))
```

Comment:
I understand you're trying here to show that the mask can be determined from the image, without prior knowledge of corrupted pixels, but it seems a bit artificial to me. Could we just have ``mask = corrupted_pixels``?

Reply:
You are right that it would be clearer. Changed